webapp/channels/src/i18n/en.json

```diff
@@ -5758,7 +5751,7 @@
   "warn_metric_ack_modal.learn_more.link": "Learn more",
   "warn_metric_ack_modal.mailto.link": "email us",
   "warn_metric_ack_modal.mailto.message": "Support could not be reached. Please {link}.",
-  "warn_metric_ack_modal.number_of_active_users.description": "Mattermost strongly recommends that deployments of over {limit} users take advantage of features such as user management, server clustering and performance monitoring. Contact us to learn more and let us know how we can help.",
+  "warn_metric_ack_modal.number_of_active_users.description": "Mattermost strongly recommends that deployments of over {limit}} users take advantage of features such as user management, server clustering, and performance monitoring. Contact us to learn more and let us know how we can help.",
```

Comment:
Are double braces needed on either side of a placeholder? ``{limit}}`` may be missing one brace.

Reply:
This seems to be an error in the code. Will fix.